src/components/VirtualList/VirtualList.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { Table, TableHeader, TableGridBreakpoint } from '@patternfly/react-table';
+import VirtualTable from './VirtualTable';
+import history from '../../app/History';
+import { config } from './Config';
+
+type VirtualListProps = {
+  rows: any[];
```

Comment:
I try to see if we can have a better typing here: as it's done with `FilterComponent`, it seems we can use generics here too:
- with here `VirtualListProps<R>` and `rows: R[];`
- on component declaration, with `class VirtualList<R extends TResource>`
- seems like in `render` we should set `tableProps.rows` to empty array instead of `rows`, which is fine as if i understand correctly, the patternfly Table component is not used to display rows but just for the header (is it correct? I tested and it seems true)